File: GGAndroid_2.0/GGDemo/src/main/java/io/ganguo/demo/viewmodel/window/TestPopupWindowVModel.java
package io.ganguo.demo.viewmodel.window;

import android.view.View;

import io.ganguo.demo.R;
import io.ganguo.demo.databinding.WindowTestPopupWindowBinding;
import io.ganguo.vmodel.BaseViewModel;
import io.ganguo.library.ui.view.PopupWindowInterface;

/**
 * <p>
 * PopupWindow - 测试Demo
 * </p>
 * Created by leo on 2018/8/2.
 */

public class TestPopupWindowVModel extends BaseViewModel<PopupWindowInterface<WindowTestPopupWindowBinding>> {
    @Override
    public int getItemLayoutId() {
        return R.layout.window_test_popup_window;
    }

    @Override
    public void onViewAttached(View view) {

    }
}

File: GGAndroid_3.0/main/app-mvvm-v1-sample/src/main/java/io/ganguo/demo/viewmodel/window/TestPopupWindowVModel.kt
package io.ganguo.demo.viewmodel.window

import android.view.View
import io.ganguo.viewmodel.core.viewinterface.WindowInterface
import io.ganguo.demo.R
import io.ganguo.demo.databinding.WindowTestPopupWindowBinding
import io.ganguo.viewmodel.core.BaseViewModel

/**
 *
 *
 * PopupWindow - 测试Demo
 *
 * Created by leo on 2018/8/2.
 */

class TestPopupWindowVModel : BaseViewModel<WindowInterface<WindowTestPopupWindowBinding>>() {

    override fun onViewAttached(view: View) {

    }

    override val layoutId: Int by lazy {
        R.layout.window_test_popup_window
    }
}
